config/locales/de.yml

```diff
@@ -41,6 +41,10 @@ de:
     index:
       lead: "Wir bieten unglaubliche Lösungen für vielfältige Use Cases."
 
+  user-dashboard:
```

Comment:
Call this navbar maybe? Hard to say whether it belongs to navbar or dashboard, just a proposal.